src/commcare_cloud/terraform/staging/terraform.tf

```diff
@@ -86,7 +86,7 @@ module "commcarehq" {
     {
       server_name           = "proxy1-staging"
       server_instance_type  = "t2.medium"
-      subnet_index          = 0
+      subnet_index          = 4
```

Comment:
I think this will conflict with your other terraform changes

Reply:
Yeah quite possibly. It'll be a very easy conflict to fix though. I'd originally done this change on this branch, but then I cherry-picked it onto https://github.com/dimagi/commcare-cloud/pull/2125 as well, after which I continued with other formatting changes to make it more readable.